### DISCUSSION TITLE
Wall mounted handheld/item holder

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4433] [Suggestions]
Credit to Aiwi for building this, he built a sort of modular wall weapon holder. Some that flip into the wall or extend out. Problem we found is that actually picking up these guns can be a challenge since the item stand is below. Also the item stand being below greatly expands the overall volume.
![image](https://user-images.githubusercontent.com/36980446/147863986-0c58e0dc-6891-41f7-a0dc-45017776a2c0.png)

Now that there are sooooo maaannnnyyyy sexy weapons built, displaying them with only the item stand can be a bit limiting. Would it be possible for there to be a stretchable wall-mounted item holder? Maybe a sort of stretchable "diorama" that has x y and z space so weapons/tools that are the right size can fit inside
![image](https://user-images.githubusercontent.com/36980446/147864014-5998a876-ed40-4e6a-9a17-7bff491e16ff.png)

**Comment by silvernull** [comment]
Im thinking
maybe just giving the item stand a variable direction would do all of this? so it could be stuck on a wall and still hold the gun in its proper orientation
